### PR TITLE
Large update to readme on clean debian 12 install

### PR DESCRIPTION
**Pull request by Oscar-Pepper**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,93 +1,126 @@
...
+   Specify to run a specific ABI and test:
+      `cargo nextest run x86_64::test_name`
+
+For more information on running integration tests on non-default AVDs, run:
+   `./scripts/integration_tests.sh -h`
+Without the cargo test runner these emulated android devices will not be able to connect to a
+lightwalletd/zcashd regtest network. Therefore, only tests in the "Offline Testsuite" may be tested.
+
+### End-to-End Tests
+1. Run `yarn react-native start` to start the dev server
```
Comment (juanky201271) [inline README.md]: Here you can put just `yarn start` as before in this document.

**Review by fluidvanadium — DISMISSED** [review]
trailblazin!

File: README.md [hunk 2]
```diff
@@ -1,93 +1,126 @@
...
+3. From the root of the project, run:
+      `yarn install`
+
+For notes on known issues and problems, see the [trouble-shooting notes](./TROUBLESHOOTING.md).
+
+## Launching the app
+1. In a terminal, run:
+      `yarn start`
+2. In a separate terminal, run:
+      `yarn run ios`
```
Comment (juanky201271) [inline README.md]: I think you don't need `run` here, just `yarn ios`.

File: README.md [hunk 3]
```diff
@@ -1,93 +1,126 @@
...
+7. Cocaopods (`sudo gem install cocoapods`)
+
+## Building
+1. In the `./rust/ios` directory, run:
+      `./build.sh`.
+   This step may take a long time.
+2. In the `./ios` directory, run:
+      `pod install`
+3. From the root of the project, run:
+      `yarn install`
```
Comment (juanky201271) [inline README.md]: In my experience the logical order of this steps is:
1. `./build.sh`
2. `yarn install` (you can omit `install` if you want to), just `yarn`
3. `pod install`, the last one.


IMO.

File: README.md [hunk 4]
```diff
@@ -1,93 +1,126 @@
...
+   export PATH=$PATH:$ANDROID_HOME/cmdline-tools/latest/bin
+   export PATH=$PATH:$ANDROID_HOME/platform-tools
+   export PATH=$PATH:$ANDROID_HOME/emulator
+
+## Building
+1. In the `rust` directory, run:
+      `./build.sh`.
+   This step may take a long time.
+2. From the root of the project, run:
+      `yarn install`
```
Comment (juanky201271) [inline README.md]: You can omit `install` if you want to, just `yarn`.

File: README.md [hunk 5]
```diff
@@ -1,93 +1,126 @@
...
+   i.e. x86_64. The recommended API is API 30 (Android 11). Alternatively, you can connect to a
+   physical device (https://reactnative.dev/docs/running-on-device).
+2. In `File > Settings`, navigate to `Build, Execution and Deployment > Build Tools > Gradle` and
+   check the `Gradle JDK` matches your JDK version.
+2. In a terminal, run:
+      `yarn start`
+3. Open the `android` directory in Android Studio as a project, select 'app' and the previously
+   created AVD in the upper toolbar and click the "Run 'app'" button.
+   Alternatively, launch an AVD and in a separate terminal, run:
+      `yarn run android`
```
Comment (juanky201271) [inline README.md]: just `yarn android`

**Comment by juanky201271** [comment]
@Oscar-Pepper 
Maybe it's worth it to add in the `readme` how to build rust lib for `IOS simulator`... it exists another script: `./rust/ios/buildsimulator.sh`. But for this we need a patch in `/rust/Cargo.toml`, like:
```
[patch.crates-io]
sodiumoxide = { git = "https://github.com/juanky201271/sodiumoxide", rev = "2f1b91e031199412ec631a3afd571a7df2450981"}
```

Because this lib is `archived` and fails building for `aarch64-apple-ios-sim`.

**Review by juanky201271 — APPROVED** [review]
terrific....

**Pull request merged**